pkg/kn/commands/subscription/flags.go

```diff
@@ -0,0 +1,137 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package subscription
+
+import (
+	"sort"
+
+	metav1beta1 "k8s.io/apimachinery/pkg/apis/meta/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"knative.dev/client/pkg/kn/commands"
+	"knative.dev/client/pkg/kn/commands/flags"
+	hprinters "knative.dev/client/pkg/printers"
+
+	messagingv1beta1 "knative.dev/eventing/pkg/apis/messaging/v1beta1"
+)
+
+// ListHandlers handles printing human readable table for `kn subscription list` command's output
+func ListHandlers(h hprinters.PrintHandler) {
+	subscriptionColumnDefinitions := []metav1beta1.TableColumnDefinition{
+		{Name: "Namespace", Type: "string", Description: "Namespace of the Subscription", Priority: 0},
+		{Name: "Name", Type: "string", Description: "Name of the Subscription", Priority: 1},
+		{Name: "Channel", Type: "string", Description: "Channel of the Subcription", Priority: 1},
+		{Name: "Subscriber", Type: "string", Description: "Subscriber of the Subscription", Priority: 1},
```

Comment:
Subscriber or subscription is a mouthful :) But not sure there is better

Reply:
thats for the list column, however while we are on it, I wanted to add an alias `subs` for `subscription` command group. I think we also had a discussion earlier to create a spreadsheet or similar document for the alias proposal? I think we should do that for the coming release ?